dashboard/common/utils.py

```diff
@@ -60,17 +61,31 @@ def search_key_for_resource_value(my_dict, search_for):
     return None
 
 
-def get_myla_globals(current_user):
+def get_course_id_from_request_url(path: str) -> Union[None, int]:
+    course_id = None
+    if settings.STUDENT_DASHBOARD_LTI:
+        # Looking for an matching pattern like this /courses/123455
+        course_id_from_path = re.findall('/courses/(\d+)\/?', path)
+        if len(course_id_from_path) == 1:
+            course_id = int(course_id_from_path[0])
+            logger.info(f'course_id from path: {course_id}')
+    return course_id
+
+
+def get_myla_globals(request):
+    current_user = request.user
     username = ""
     user_courses_info = []
     login_url = ""
     logout_url = ""
     google_analytics_id = ""
+    path = request.path
```

Comment:
nitpick: There isn't really any value to having path be assigned here as it's only used once. You can just remove this and use request.path as the parameter.

Reply:
Done